app/services/control_rate_service.rb

```diff
@@ -41,80 +41,120 @@ def all_cached_data
 
   def fetch_all_data
     results.registrations = registration_counts
-    results.registrations_with_exclusions = registration_counts_with_exclusions
+    results.denominator_registrations = denominator_registration_counts
     results.earliest_registration_period = registration_counts.keys.first
+    results.full_data_range.each do |(period, count)|
+      results.ltfu_patients[period] = ltfu_patients(period).count
+    end
     results.fill_in_nil_registrations
     results.count_cumulative_registrations
-    results.count_adjusted_registrations
+    results.count_adjusted_registrations_with_ltfu
+
+    if with_exclusions
+      results.count_adjusted_registrations
+    else
+      results.adjusted_registrations = results.adjusted_registrations_with_ltfu
+    end
```

Comment:
The adjusted registrations are defaulted to `adjusted_registrations_with_ltfu` if the flipper flag is off.